snap/snapcraft.yaml

```
@@ -0,0 +1,24 @@
+name: sabnzbd
+version: git
+summary: SABnzbd
+description: The automated Usenet download tool
+grade: stable
+confinement: strict
+
+apps:
+  sabnzbd:
+    command: opt/sabnzbd/SABnzbd.py -f $SNAP_COMMON
```

```suggestion
    command: python $SNAP/opt/sabnzbd/SABnzbd.py -f $SNAP_COMMON
```

Combine this with reverting the `#!` line in `SABnzbd.py`.